### PR TITLE
Syndicate Bombs were Intended to Be CanCollide

### PR DESCRIPTION
**Pull request by gusxyz**
## About the PR
Discussed with rain they were meant to be can collide. 
Its intended to spawn it unanchored, it wont get you stuck now

:cl: gusxyz
- tweak: Syndicate bombs collide again.

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
you forgot `ed`

**Comment by gusxyz** [comment]
> you forgot `ed`

💀

**Comment by metalgearsloth** [comment]
Why is this static but not anchored?

**Comment by gusxyz** [comment]
its dynamic and removed noRot

**Comment by Partmedia** [comment]
This kind of behaves weirdly when it's noRot but still allowed to rotate, e.g. taking this and pushing it into a wall causes to rotate, with no way to rotate it back.

image

**Comment by gusxyz** [comment]
added rotatable tag

File: Resources/Prototypes/Entities/Structures/Machines/bombs.yml [hunk 1]
```diff
@@ -32,10 +33,9 @@
     - type: Anchorable
       delay: 5
     - type: Physics
-      bodyType: Static
-      canCollide: false
+      bodyType: Dynamic
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Structures/Machines/bombs.yml:36]: This duplicates what's already in `BaseStructureDynamic`.

File: Resources/Prototypes/Entities/Structures/Machines/bombs.yml [hunk 2]
```diff
@@ -32,10 +33,9 @@
...
     - type: Transform
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Structures/Machines/bombs.yml:37]: This also duplicates what's already in `BaseStructureDynamic`.

**Review by Partmedia — CHANGES_REQUESTED** [review]
Sorry for the delay on this. Please see the review comments.

**Comment by Partmedia** [comment]
Now the bomb doesn't snap itself to a cardinal direction when it anchors itself, but I'm fine with this.
image

**Comment by Partmedia** [comment]
What's weird is that when you anchor it with a wrench, it snaps correctly, but it doesn't snap when you begin the countdown, hmm...

**Comment by Partmedia** [comment]
The bug is that `TryStartCountdown` uses `_transform.AnchorEntity()` instead of going through AnchorableSystem's `OnAnchorComplete()` which handles the cardinal direction snapping. Since this is not your bug, I'm going to go ahead and merge this. @Just-a-Unity-Dev

**Review by Partmedia — APPROVED** [review]

**Comment by Just-a-Unity-Dev** [comment]
> The bug is that `TryStartCountdown` uses `_transform.AnchorEntity()` instead of going through AnchorableSystem's `OnAnchorComplete()` which handles the cardinal direction snapping. Since this is not your bug, I'm going to go ahead and merge this. @Just-a-Unity-Dev 

a

https://github.com/space-wizards/space-station-14/assets/67359748/9313edf8-5626-4052-b7fd-59634d7b508e

I'll try to fix it within this week, but I've hit a motivation block

**Pull request merged**